modules/cnf-image-based-upgrade-rollback.adoc

```diff
@@ -32,6 +32,11 @@ You can manually roll back the changes if you encounter unresolvable issues afte
 * You have logged into the hub cluster as a user with `cluster-admin` privileges.
 * You ensured that the control plane certificates on the original stateroot are valid. If the certificates expired, see "Recovering from expired control plane certificates".
 
+[WARNING]
+====
+If you choose to upgrade a recently installed {sno} cluster for testing purposes, you have a limited rollback time frame of 24 hours or less. You can verify the rollback time by checking the `rollbackAvailabilityExpiration` field of the image-based upgrade custom resource.
```

Comment:
"for testing purposes" is redundant.
What if I decide to upgrade a recently deployed cluster for non-testing purposes, for example because I realized there's an upgrade available and it's easier for me to upgrade instead of re-installing? Is that a non-valid scenario?